Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerController : MonoBehaviour {
+
+  public float Speed;
+
+  private Rigidbody rigidbody;
```

Comment:
[MUST]
`rigidbody` is duplicated the same name in MonoBehaviour's members although witch was obsoleted.
You should use a different name.